public/views/tour.html

```diff
@@ -0,0 +1,65 @@
+<div id="app-container" ng-controller="GameController">
+  <header>
+    <div class="navbar-fixed">
+      <nav class="main-header white cfh-logo-header">
+        <div class="container nav-wrapper">
+          <a href="index.html" class="cfh-logo black-text">
+            <img src="img/cfh-logo.png" alt="cfh-logo"/>
+          </a>
+          <a icon-label="mobile-menu" class="nav-mobile">
+            <span></span>
+            <span></span>
+            <span></span>
+          </a>
+          <ul class="nav-list right hide-on-med-and-down">
+            <li class="nav-item"><a href="#home">Home</a></li>
+            <li class="nav-item"><a href="#how-it-works">How it works</a></li>
+            <li class="nav-item"><a href="#login">Login</a></li>
+            <li class="nav-item"><a href="#signup">Signup</a></li>
+            <li class="nav-item">
+              <a href="#donate">
+                <button class="btn brand-btn brand-btn-donate">
+                  Donate
+                </button>
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="mobile-nav">
+          <div class="container" id="mobile-nav"></div>
+        </div>
+      </nav>
+    </div>
+  </header>
+  <div class="row">
+    <div class="container">
+      <div class="control-game">
+        <span id="tweet-container">
+          <a href="https://twitter.com/share" class="twitter-share-button" data-url="http://cfh.io" data-text="Cards for Humanity: A Game for Horrible People Desperately Trying to do Good" data-related="CFH_App">Tweet</a>
+        <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
+        </span>
+        <button id="abandon-game-button" class="btn red darken-4 right" ng-click="abandonGame()">
+          Abandon Game
```

Comment:
I thought this has been changed to `Leave Game`?

Reply:
Yes. It has. I'll correct that